ruby-training/trainees/alan/test_katas/05/Sum_with_out_max_min/sum_with_out_max_min.rb

```diff
@@ -0,0 +1,18 @@
+
+class SumWithOutMaxMin
+    
+    def self.sort_array_of_intergers(array_of_intergers)
```

Comment:
You could probably remove this method, and just call `array_of_integers.sort` from `eliminate_min_max` instead.